contrib/datawave-quickstart/bin/services/hadoop/install.sh

```diff
@@ -89,13 +89,10 @@ else
 fi
 
 # Ensure that $JAVA_HOME is observed by all hadoop scripts
-sed -i "s|.*\(export JAVA_HOME=\).*|\1${JAVA_HOME}|g" ${HADOOP_CONF_DIR}/hadoop-env.sh
+sed -i'.bak' "s|.*\(export JAVA_HOME=\).*|\1${JAVA_HOME}|g" ${HADOOP_CONF_DIR}/hadoop-env.sh && rm ${HADOOP_CONF_DIR}/hadoop-env.sh.bak
```

Comment:
Why not just `sed -i ''` instead of `sed -i '.bak'` and then you don't have to remove the `.bak` file separately?